src/docs/Components/Form/Radio/index.mdx

```diff
@@ -11,6 +11,67 @@ status:
 
 <Preview path="intro" nude />
 
+
+## Overview
+
+<Highlight theme="warning">
+
+  A radio button can't be used alone, it must be used within a group.
+
+</Highlight>
+
+![Radio-Overview](Radio-Overview.png)
+
+Checkbox group is a combination of the following mentions. Please note that you can use four different cases:
+  - Group label + radio buttons
+  - Group label + requirement + radio buttons
+  - Group label + requirement + help text + radio buttons
+  - Groupe label + help text + radio buttons
+
+### Label group
+
+The label group helps the user to understand which radio to check.
+
+<Preview path="group-default" nude />
+
+### Requirement
+
+This mention allows to specify that this information is mandatory for the input to be valid.
+
+<Highlight theme="tips">
+
+  Please note that this mention can also be used to say a specific input is **optional** when the other ones are mandatory.
+
+</Highlight>
+
+<Preview path="group-requirement-light" nude />
+
+### Help text
+
+Help text corresponds to a relevant piece of information to make sure the user understands what is needed.
+
+<Preview path="group-help-light" nude />
+
+## Layouts
+
+### Vertical group
+
+<Preview path="group-default" nude />
+
+### Horizontal group
+
+<Preview path="group-inline" nude />
+
+## Validation
+
+There are two different elements for validation feedback:
+  - The color of the input outline.
+  - A text under the radio button group explains what's wrong with the data entered.
+
+<Preview path="group-invalid-light" nude />
+
+
+
 ## Do & dont's
 
```

Comment:
Never use the radio alone. You should always use it with a label and within a group.

Reply:
Fixed